### PR TITLE
Refactor unused code & transformation

### PR DESCRIPTION
**Pull request by vitaliystoliarovcc**
Asana ticket: https://app.asana.com/0/1125859137800433/1200420964765202/f

- Removed unused JS modules
- Enum WS connection state
- Refactored Bitfinex exchange client's data transformer

### CODE REVIEW

File: lib/ws_servers/api/send_balances.js [hunk 1]
```diff
@@ -1,9 +0,0 @@
-'use strict'
-
-const send = require('../../util/ws/send')
-
-// TODO: Implement for bfx
-module.exports = async (ws, exID, exClient) => {
-  const balances = await exClient.getBalances()
-  send(ws, ['data.balances', exID, balances])
-}
```
Comment (robertkowalski) [inline lib/ws_servers/api/send_balances.js:9]: nice!

File: lib/exchange_clients/bitfinex/recv/data.js [hunk 2]
```diff
@@ -20,7 +22,7 @@ module.exports = (exa, msg) => {
   // give subscription promises a chance to resolve
   setTimeout(() => {
     for (let i = 0; i < dataListeners.length; i += 1) {
-      dataListeners[i](chanID, msg)
+      dataListeners[i]([msgType, transform(msgType, msgData)])
     }
   }, 0)
```
Comment (robertkowalski) [inline lib/exchange_clients/bitfinex/recv/data.js:27]: maybe we can use `setImmediate` or `nextTick` instead?

Comment (vitaliystoliarovcc) [reply lib/exchange_clients/bitfinex/recv/data.js:27]: I'm not sure if this won't break anything since `setImmediate`  and `nextTick` works differently

**Review by avsek477 — APPROVED** [review]
lgtm

**Pull request merged**